aws/hll/aws-s3-transfer-manager/src/download.rs

```diff
@@ -34,17 +49,178 @@ impl From<GetObjectInputBuilder> for DownloadRequest {
     }
 }
 
-/// Response type for a single download object request.
-#[derive(Debug)]
-#[non_exhaustive]
-pub struct DownloadResponse {
-    /// Object metadata
-    pub object_meta: ObjectMetadata,
+/// Fluent style builder for [Downloader]
+#[derive(Debug, Clone)]
+pub struct Builder {
+    target_part_size_bytes: u64,
+    checksum_validation_enabled: bool,
+    // TODO(design): should we instead consider an enum here allows for not only explicit but also
+    // an "Auto" mode that allows us to control the concurrency actually used based on overall transfer and part size?
+    concurrency: usize,
```

Comment:
I think yes, but it doesn't matter unless we have to expose this. Unexposed, we're free to change the impl whenever.